### PR TITLE
two-fer: Create exercise

### PR DESCRIPTION
**Pull request by sshine**
* Add README.md from canonical data.
* Base exercise file structure on hello-world (#5).

### CODE REVIEW

**Comment by sshine** [comment]
@glennj: I've made a lot of assumptions taking inspiration from various other tracks I've been involved with. They're in no way set in stone, and I haven't taken the time to document them yet; these are just bits I had lying around waiting for myself to submit initial CI scripts (#8).

**Comment by sshine** [comment]
Yes, the format and placement of the test version are two such assumptions.

I've opened an issue to discuss this: #10.

**Comment by sshine** [comment]
I'm a little curious as to why [my own CI run](https://travis-ci.org/sshine/exercism-tcl/builds/568350847) fails with "The exercise 'two-fer' is marked as core and unlocked by another exercise. A core exercise should not be unlocked by another." when this PR's CI run didn't. Maybe I forgot the order in which I merged the CI commit and my latest push to this PR. Never mind. I've fixed the bug.

**Comment by glennj** [comment]
I did notice the core and unlocked. I guess we've confirmed that you can't have both.

File: exercises/two-fer/test.tcl [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        No name given
+
+        Test [two-fer] == "One for you, one for me."
+
+        It should be possible to call [two-fer] without arguments.
+    } -body {two-fer} -result "One for you, one for me."
+    test two-fer "A name given" -body {two-fer "Alice"} -result "One for Alice, one for me."
+    test two-fer "Another name given" -body {two-fer "Bob"} -result "One for Bob, one for me."
+    cleanupTests
+}
```
Comment (glennj) [inline exercises/two-fer/test.tcl]: I find this really hard to read. Can we establish some code convention for test files? 
* whitespace between tests
* always add newlines and indentation for body code

Comment (sshine) [reply exercises/two-fer/test.tcl]: I've done those, and I've changed the descriptions to be exactly the canonical descriptions and nothing else. (The `Test [two-fer] == "One for you, one for me."` was something I'd seen in a test online when reading about the test framework, but the precise command being tested is actually printed, so this is superfluous.

**Comment by sshine** [comment]
I've made this PR into a WIP one because:
- I'd like to make sure that we use a squash commit in this case.
- I'd like to make sure that `./test.tcl` will give a non-zero exit code cf. #11.

**Pull request merged**